cmd/kots/cli/backup-create.go

```diff
@@ -0,0 +1,38 @@
+package cli
+
+import (
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kots/pkg/snapshot"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func BackupCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "create",
+		Short:         "Creates an instance backup/snapshot",
+		Long:          ``,
+		SilenceUsage:  true,
+		SilenceErrors: false,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			viper.BindPFlags(cmd.Flags())
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			v := viper.GetViper()
+
+			instanceBackupOptions := snapshot.InstanceBackupOptions{
+				Namespace:             v.GetString("namespace"),
+				KubernetesConfigFlags: kubernetesConfigFlags,
+			}
+			if err := snapshot.InstanceBackup(instanceBackupOptions); err != nil {
+				return errors.Cause(err)
```

Comment:
The problem with using `Cause` here is that it could return something like "EOF" without any other context.  If there are some human readable errors, we could create a type of those and strip the error stack for all other types.